module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FavoritesTest.php

```diff
@@ -707,12 +707,12 @@ public function testPublicListIndicator(): void
         // The "Future List" should NOT be public:
         $this->clickCss($page, 'a[href="' . $hrefs[0] . '"]');
         $this->snooze();
-        $this->assertEquals(0, count($page->findAll('css', 'strong i.fa-globe')));
+        $this->assertEquals(0, count($page->findAll('css', 'strong .fa-globe')));
 
         // The "Test List" SHOULD be public:
         $this->clickCss($page, 'a[href="' . $hrefs[2] . '"]');
         $this->snooze();
-        $this->assertEquals(1, count($page->findAll('css', 'strong i.fa-globe')));
+        $this->assertEquals(1, count($page->findAll('css', 'strong .fa-globe')));
```

Comment:
It might be worth removing all icon names and classes from the integration tests.

Reply:
Agreed -- if you apply classes based on the abstract, VuFind-specific icon names instead of icon-library-specific details, testing for those would be a lot more future-proof.